README.md

```diff
@@ -203,7 +203,24 @@ Note that when no value is given the progress is auto-detected baed on the envir
 
 #### Prerequisites
 
-In order to successfully import an AMI into your AWS account, you need to have the [vmimport service role](https://docs.aws.amazon.com/vm-import/latest/userguide/required-permissions.html) configured on your account.
+In order to successfully import an AMI into your AWS account, you need to have the [vmimport service role](https://docs.aws.amazon.com/vm-import/latest/userguide/required-permissions.html) configured on your account with the following additional permissions:
+
+```
+{
+    "Effect": "Allow",
+    "Action": [
+        "s3:ListAllMyBuckets",
+        "s3:GetBucketAcl",
+        "s3:DeleteObject"
+    ],
+    "Resource": [
+        "arn:aws:s3:::amzn-s3-demo-import-bucket",
+        "arn:aws:s3:::amzn-s3-demo-import-bucket/*",
+        "arn:aws:s3:::amzn-s3-demo-export-bucket",
+        "arn:aws:s3:::amzn-s3-demo-export-bucket/*"
```

Comment:
I saw the word `demo` in the resource names and had to look up how they're used exactly.  We should mention in the text above that `amzn-s3-demo-import-bucket` and `amzn-s3-demo-export-bucket` are example names.  They are, right?

It should also be enough to just have one, maybe?  Or change the name to reflect our use case as a slightly more concrete example.  Like `bib-ec2-image-upload-bucket-example` or something.

Or am I wrong about what this all means?

Reply:
This is really the bucket name in ARN format, I was trying to match what is documented in the linked AWS doc rather than this README. I can change if you insist, I take it since this is part of the documentation it should be more clear to a user who is copy-pasting these blocks into AWS UI that they should replace this one as well.

Reply:
Yeah, the example names don't need to change, but a mention that they should be replaced by the bucket name would be good.

Reply:
Oh, of course, rebased.